src/renderer/fonts/font_options.rs

```diff
@@ -204,7 +204,12 @@ impl FontOptions {
             .map(|fonts| {
                 fonts
                     .iter()
-                    .flat_map(|font| font.fallback(&self.normal))
+                    .filter(|font| font.family.is_some())
+                    .map(|font| FontDescription {
+                        family: font.family.clone().unwrap(),
+                        style: None,
+                    })
+                    .chain(self.normal.iter().cloned())
```

Comment:
Should we respect users normal weight?
eg.
```toml
[[font.normal]]
family = "not found1"
style = "W400"

[[font.normal]]
family = "menlo"
style = "italic"

[[font.bold]]
family = "something"
style = "W600"
```
Should this force normal's weight to Bold or use italic?

Reply:
I think we should force the weight to bold. That's more simple, and people who need special fallbacks and weights can always manually define fallbacks for all styles.